src/dotnet/UI.Blazor/Components/Separator/BadgeSeparator.razor

```diff
@@ -0,0 +1,11 @@
+@namespace ActualChat.UI.Blazor.Components
+
+<div class="@Class badge-separator">
```

Comment:
Why components is called `BadgeSeparator`?

Reply:
it looks like separator with badge